### DISCUSSION TITLE
point to non-public schema in Postgres self-host?

### DISCUSSION THREAD

**Discussion by P01NTE45T** [FreshRSS/FreshRSS#5998] [Q&A] [answered]
Hi,

Wondering if I can point to a schema other than "public" in my self-hosted Postgresql Database during configuration?

**Comment by Alkarex** [comment]
I have not tested, but what have you tried? Have you tried the dot syntax for instance `something.database` in the database name?

**Reply by P01NTE45T** [reply]
Many thanks for swift response. I have, yes. Also ":", "@", etc.

**Reply by P01NTE45T** [reply, marked as answer]
I found a workaround by assigning the default schema to the freshrss user in postgresql.
